src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+`;
+
+const TooltipArrowBase = styled.div`
+  border-color: transparent;
+  border-style: solid;
+  height: 0;
+  position: absolute;
+  width: 0;
+`;
+
+const TooltipTop = styled(TooltipBase)`
+  margin-top: -3px;
+  padding: 5px 0;
+`;
+
+const TooltipRight = styled(TooltipBase)`
+  margin-right: 3px;
+  padding: 0 5px;
+`;
+
+const TooltipBottom = styled(TooltipBase)`
+  margin-bottom: 3px;
+  padding: 5px 0;
+`;
+
+const TooltipLeft = styled(TooltipBase)`
+  margin-left: -3px;
+  padding: 0 5px;
+`;
+
+const TooltipArrowTop = styled(TooltipArrowBase)`
+  border-top-color: ${colors.black};
+  border-width: 5px 5px 0;
+  bottom: 0;
+  left: 50%;
+  margin-left: -5px;
+`;
+
+const TooltipArrowRight = styled(TooltipArrowBase)`
+  border-right-color: ${colors.black};
+  border-width: 5px 5px 5px 0;
+  left: 0;
+  margin-top: -5px;
+  top: 50%;
+`;
+
+const TooltipArrowBottom = styled(TooltipArrowBase)`
+  border-bottom-color: ${colors.black};
+  border-width: 0 5px 5px;
+  left: 50%;
+  margin-left: -5px;
+  top: 0;
+`;
+
+const TooltipArrowLeft = styled(TooltipArrowBase)`
+  border-left-color: ${colors.black};
+  border-width: 5px 0 5px 5px;
+  margin-top: -5px;
+  right: 0;
+  top: 50%;
+`;
+
+const Fade = (props) => (
+  <Transition
+    className="transition-out"
+    enteredClassName="transition-in"
+    enteringClassName="transition-in"
+    in={props.in}
+    mountOnEnter="true"
+    unmountOnExit="false"
+    timeout="200"
+    transitionAppear="true"
+  >
+    {props.children}
+  </Transition>
+);
+
+Fade.propTypes = {
+  children: PropTypes.any.isRequired,
+  in: PropTypes.boolean
+};
+
+const Popup = (props) => {
+  const { className, children, position, style } = props;
+  let TooltipStyled;
+  let TooltipArrow;
+
+  switch (position) {
+    case 'right':
+      TooltipStyled = TooltipRight;
+      TooltipArrow = TooltipArrowRight;
+      break;
+    case 'bottom':
+      TooltipStyled = TooltipBottom;
+      TooltipArrow = TooltipArrowBottom;
+      break;
+    case 'left':
+      TooltipStyled = TooltipLeft;
+      TooltipArrow = TooltipArrowLeft;
+      break;
+    default:
+      TooltipStyled = TooltipTop;
+      TooltipArrow = TooltipArrowTop;
+      break;
+  }
+
+  return (
+    <TooltipStyled className={className} style={{ ...style }}>
+      <TooltipArrow />
+      <TooltipInner>
+        {children}
+      </TooltipInner>
+    </TooltipStyled>
+  );
+};
+
+Popup.propTypes = {
+  children: PropTypes.any.isRequired,
+  className: PropTypes.string,
+  position: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
+  style: PropTypes.object
+};
+
+class Tooltip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { show: false };
+  }
+
+  show() {
+    return this.setState({ show: true });
+  }
+
+  hide() {
+    return this.setState({ show: false });
+  }
+
+  render() {
+    return (
+      <span>
+        <span ref="target" onMouseEnter={() => this.show()} onMouseLeave={() => this.hide()}>
```

Comment:
Let's rethink how we're setting up the ref here. using string refs is the [legacy way](https://facebook.github.io/react/docs/refs-and-the-dom.html#legacy-api-string-refs) of doing it. You should probably set up the ref in [component did mount](https://facebook.github.io/react/docs/refs-and-the-dom.html#adding-a-ref-to-a-class-component)

Using this will also allow you to get rid of the findDOMNode dependency.